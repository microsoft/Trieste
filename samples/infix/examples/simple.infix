x = 5;
print "x" x;
y = 2 - 1;
print "1 + 10" 1 + 10;
