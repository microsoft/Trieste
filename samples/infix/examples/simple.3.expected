//! postfix
//! --enable-tuples postfix
//! --enable-tuples --tuples-require-parens postfix
//! --enable-tuples --tuples-require-parens --use-parser-tuples postfix
x 5 =
"x" x print
y 2 1 - =
"1 + 10" 1 10 + print
